StandardAnalysis/python/customize.py

```diff
@@ -193,6 +194,13 @@ def customize (process,
         process.TriggerWeightProducer.produceTrackLeg = cms.bool(False)
         process.TriggerWeightProducer.produceGrandOr = cms.bool(True)
 
+        if dataEra in ['E', 'F', 'G']:
+            process.EventJetVarProducer.jetVetoMap = cms.FileInPath ('OSUT3Analysis/Configuration/data/Summer22EE_23Sep2023_RunEFG_v1.root')
+            print("Using jet veto map for 2022 eras E/F/G")
+        else:
+            process.EventJetVarProducer.jetVetoMap = cms.FileInPath ('OSUT3Analysis/Configuration/data/Summer22_23Sep2023_RunCD_v1.root')
+            print("Using jet veto map for 2022 eras C/D")
+
```

Comment:
I would suggest to include the `dataYear` as a condition as well, because there might be different maps for 2023, right? Even though it is not going to be used right now, it would be nice to have it there as a reminder.

Reply:
the year already has its own section as an input to customize that is set in the config_2022_cfg file. I agree is is a bit confusing having this set in multiple places. Maybe we can remove it from proto_config and make it an input only to customize?

Reply:
Ah, alright! That is indeed confusing.

About customize, we already have to keep changing the year for each customization. Maybe adding an era change won't be too much. I think we can add another variable to the customize function that controls the era. And then, in the `config_dataYear_cfg.py`, the user can choose the year/era to apply the correct customization. Does this sounds reasonable?